x-pack/legacy/plugins/canvas/public/lib/template_from_react_component.tsx

```diff
@@ -4,15 +4,24 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
-import ReactDom from 'react-dom';
+import React, { ComponentType, FunctionComponent } from 'react';
+import ReactDom, { unmountComponentAtNode } from 'react-dom';
```

Comment:
Just curious: Is there a reason you destructure `unmountComponentAtNode` but not say `render` or whatever else is used from ReactDom?